src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -45,7 +45,7 @@ PylonCameraParameter::PylonCameraParameter() :
         binning_y_given_(false),
         downsampling_factor_exp_search_(1),
```

Comment:
Can we get rid of this `PylonCameraParameter` class completely and replace it with the one that dynamic-reconfigure generates automatically? That will require adding the remaining parameters to dynamic_reconfigure (e.g. `mtu_size`), but that will be a good idea anyway.